doc/api/child_process.md

```diff
@@ -868,7 +868,8 @@ changes:
   * `output` {Array} Array of results from stdio output.
   * `stdout` {Buffer|string} The contents of `output[1]`.
   * `stderr` {Buffer|string} The contents of `output[2]`.
-  * `status` {number} The exit code of the child process.
+  * `status` {number|null} The exit code of the subprocess, or `null` if the
+    child process exited due to a signal.
```

Comment:
"exited due to a signal" --> "terminated due to a signal" (if it terminated due to a signal, it didn't exit, which is why there is no exit code).

the signal docs below have the same problem, they should be changed to something like

> The signal that terminated the child process, or null if the child process exited.

(or "if the child process terminated due to exiting.", but that is maybe too verbose).

Perhaps its obvious from the above, but you may want to emphasize that either status will be non-null, or signal will be non-null, but they will NEVER both have non-null values.

Reply:
When talking about processes, I tend to only use `terminate` as a transitive verb ("terminate the process" and not "the process terminated"). I'm hesitant to mix in the intransitive sense like this, but I suppose it's clear enough. Leaving this here in case someone is all "No! Don't do that! Use it just one way or the other!" @nodejs/documentation

Reply:
Meh, that's me being goofy. I wouldn't hesitate to mix transitive and intransitive uses of `stop`. I'm going to delete this and my above comment.

Reply:
@sam-github OK, I *think* I've addressed this. I went with `end due to signal` instead of `terminate due to signal`. PTAL.